src/style.css

```diff
@@ -0,0 +1,139 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  --white-color: #fff;
+  --border-color: #f3f3f3;
+  --title-color: #060b35;
+  --subtitle-color: #616070;
```

Comment:
move variables to `:root` pseudo selector

Reply:
You have these variables on a separate file - you need to remove them from here.

Reply:
but this variables already removed and located in variables.css file,
so I don't understand why the style.css has been uploaded to the github again